avalon/maya/pipeline.py

```diff
@@ -13,8 +13,10 @@
 from ..vendor.Qt import QtCore, QtWidgets
 
 # Backwards compatibility
-from .compat import load, update, remove
-from ..pipeline import create
+load = compat.load
```

Comment:
Love it. Thanks!

Reply:
I would go further actually.

Have a look a [this](https://github.com/pyblish/pyblish-base/blob/master/pyblish/compat.py#L109).

It's got two advantages. 

1. It doesn't pollute readability.
2. *Not* importing this module means cleanly cutting backwards compatibility.

Because it's backwards compatibility, there is rarely if ever a need to modify the members in this module, so it's safe to make more difficult to read, grok and edit.